### PR TITLE
[RFC][UN-15257] Bundle ember-cli-uniq dependencies inside package

### PR DESCRIPTION
**Pull request by migueldemoura**
We could also have these on a blueprint so they get added on ember install, but since this project is under active development I think this is easier. Thoughts?

Fixes:
https://github.com/uniplaces/ember-cli-uniq/issues/208
https://github.com/uniplaces/ember-cli-uniq/issues/265

### CODE REVIEW

**Review by mmelo — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -29,7 +29,10 @@
     "ember-cli-babel": "^7.1.3",
     "ember-cli-htmlbars": "^3.0.1",
     "ember-cli-sass": "^8.0.1",
-    "uniplaces-uniq": "^2.5.3"
+    "ember-progress-bar": "^1.0.0",
+    "ember-responsive": "^3.0.0-beta.3",
```
Comment (rntdrts) [inline package.json]: 🤔

File: package.json [hunk 2]
```diff
@@ -29,7 +29,10 @@
     "ember-cli-babel": "^7.1.3",
     "ember-cli-htmlbars": "^3.0.1",
     "ember-cli-sass": "^8.0.1",
-    "uniplaces-uniq": "^2.5.3"
+    "ember-progress-bar": "^1.0.0",
```
Comment (AndreJoaquim) [inline package.json]: This is only necessary if you use the `uni-progress-bar` which many projects don't. Therefore, this should not be here

**Review by AndreJoaquim — CHANGES_REQUESTED** [review]

**Review by rntdrts — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (migueldemoura) [reply package.json]: That is true but also applies to other dependencies. I think this should be addressed by https://github.com/uniplaces/ember-cli-uniq/issues/279 instead, especially given your own comment at https://github.com/uniplaces/ember-cli-uniq/issues/208.

**Comment by tlfbrito** [comment]
@migueldemoura please sync with @AndreJoaquim about this

File: package.json (hunk 2 above)
Comment (migueldemoura) [reply package.json]: After taking another look, it seems we only use it on `uni-photo-thumbnail`, a component too domain specific that arguably shouldn't even be on ember-cli-uniq anyway. I'll remove the dependency and add a note to remove this in the future.

**Review by tlfbrito — APPROVED** [review]

**Review by AndreJoaquim — APPROVED** [review]

**Pull request merged**